terraform/state_bucket/bucket.tf

```diff
@@ -0,0 +1,51 @@
+resource "aws_s3_bucket" "bucket" {
+  bucket = "osquery-foundation-terraform-state"
+  acl    = "private"
+  region = "eu-west-2" 
+
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm = "AES256"
+      }
+    }
+  }
+
+  logging {
+    target_bucket = "bucket_to_go_here"
```

Comment:
What org would we want to land this in, and can it also be driven by this TF? 

(not sure if it would want to be a variable)

Reply:
So there is a limitation of bucket logging that it has to be in the same Account and Region as the AWS project the original project is used in.